### DISCUSSION TITLE
No rule to make target 'src/modules/example.o', needed by 'src/modules/built-in.o'.

### DISCUSSION THREAD

**Discussion by Simply-Spinach** [bitcraze/discussions#1709] [Q&A]
I'm trying to follow this tutorial ( https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/development/systemtask/ ) on the crazyflie website to create a simple hello world output from the crazyflie.  I am getting the error though when I try to 'make clean && make' where it stops after building the src/modules.src/built-in.o and gives the following errors:
     make[3]: *** No rule to make target 'src/modules/example.o', needed by 'src/modules/built-in.o'.  Stop.
     make[2]: *** [../scripts/Makefile.build:311: src/modules] Error 2
     make[1]: *** [tools/kbuild/Makefile.kbuild:587: src] Error 2
     make[1]: Leaving directory '/home/cael/crazyflie-firmware/build'
     make: *** [tools/kbuild/Makefile.kbuild:150: sub-make] Error 2
I'm new to programming the crazyflie and messing with makefiles so any help is greatly appreciated.

**Comment by Simply-Spinach** [comment]
Found the problem.  Remember when you are adding the task to the correct kbuild file.  It should be in src/modules/src/ and not src/modules (it will appear in the middle of all the .c files in Visual Studio for some reason).

**Reply by ToveRumar** [reply]
Great that you found a solution to your problem!
